compiler/noirc_evaluator/src/brillig/brillig_ir/procedures/mod.rs

```diff
@@ -103,7 +103,7 @@ impl std::fmt::Display for ProcedureId {
             ProcedureId::PrepareVectorInsert => write!(f, "PrepareVectorInsert"),
             ProcedureId::VectorRemove => write!(f, "VectorRemove"),
             ProcedureId::CheckMaxStackDepth => write!(f, "CheckMaxStackDepth"),
-            ProcedureId::RevertWithString(_) => write!(f, "RevertWithString"),
+            ProcedureId::ErrorWithString(_) => write!(f, "RevertWithString"),
```

Comment:
Do you think it will be confusing to see `RevertWithString` in `--show-brillig` and then a different enum name in the code?